working/en/index.html

```diff
@@ -0,0 +1,412 @@
+												</div>
+											</div>
+										</div>
+									</div>
+									<div class="wpb_column vc_column_container vc_col-sm-2">
+										<div class="vc_column-inner">
+											<div class="wpb_wrapper">
+												<div class="content-title-element white size2">
+													<div class="title">Menu </div>
+												</div>
+												<div class="vc_empty_space"   style="height: 25px" ><span class="vc_empty_space_inner"></span></div>
+													<div  class="vc_wp_custommenu wpb_content_element">
+														<div class="widget widget_nav_menu">
+															<div class="menu-main-menu-container">
+																<ul id="menu-main-menu-2" class="menu">
+																	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-home menu-item-30">
+																		<a href="./index.html">Home</a>
+																	</li>
+																	<li class="menu-item menu-item-type-post_type menu-item-object-page current-menu-item page_item page-item-14 current_page_item menu-item-24">
+																		<a href="./team.html" aria-current="page">Team</a>
+																	</li>
+																	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-465">
+																		<a href="http://onelink.to/8hhmyv">YouClap App</a>
+																	</li>
+																</ul>
+															</div>
+														</div>
+													</div>
+												</div>
+											</div>
+										</div>
+										<div class="wpb_column vc_column_container vc_col-sm-2">
+											<div class="vc_column-inner">
+												<div class="wpb_wrapper">
+													<div class="content-title-element white size2">
+														<div class="title">Follow us </div>
+													</div>
+													<div class="vc_empty_space"   style="height: 25px" ><span class="vc_empty_space_inner"></span></div>
+													<div class="footer-social">
+														<a href="https://www.facebook.com/getYouClap/" class="facebook" title="Facebook" target="_blank">
+															<i class="fa fa-facebook"></i>
+														</a>
+														<a href="https://twitter.com/getYouClap" class="twitter" title="Twitter" target="_blank">
+															<i class="fa fa-twitter"></i>
+														</a>
+														<a href="https://www.instagram.com/getyouclap/" class="instagram" title="Instagram" target="_blank">
+															<i class="fa fa-instagram"></i>
+														</a>
+														<a href="https://www.twitch.tv/youclap" class="twitch" title="Twitch" target="_blank">
+															<i class="fa fa-twitch"></i>
+														</a>
+													</div>
+												</div>
+											</div>
+										</div>
+									</div>
+								</div>
+							</p>
+						</div>
+					</div>
+				</footer>							
+			</div>
+							
+	
+	<script type='text/javascript' src='./../assets/js/bootstrap.min.js?ver=5.1'></script>
+	<script type='text/javascript' src='./../assets/js/esport.js?ver=5.1'></script>
+	</body>
```

Comment:
You should add google analytics in all files...

Reply:
loading bootstrap in the end of the file doesn't seems a good idea 🤔